test/unit/gateways/plexo_test.rb

```diff
@@ -329,6 +338,20 @@ def test_scrub
     assert_equal @gateway.scrub(pre_scrubbed), post_scrubbed
   end
 
+  def test_network_tokenization_success
+    purchase = stub_comms do
+      @gateway.purchase(@amount, @network_token_credit_card, @options)
+    end.check_request do |_endpoint, data, _headers|
+      request = JSON.parse(data)
+      assert_equal request['Amount']['Currency'], 'UYU'
+      assert_equal request['Amount']['Details']['TipAmount'], '5'
+      assert_equal request['Flow'], 'direct'
```

Comment:
💬 Comment:

here you are not testing the change on the behavior that this PR introduces, the only think that I can image to test is the inclusion of the  `Cryptogram` parameter to ensure that your change actually modifies the request.